src/assets/scripts/client/parsers/aircraftCommandParser/AircraftCommandParser.js

```diff
@@ -323,7 +305,12 @@ export default class AircraftCommandParser {
      * @return {boolean}
      */
     _isSystemCommand(callsignOrSystemCommandName) {
-        return _has(SYSTEM_COMMANDS, callsignOrSystemCommandName) &&
-            callsignOrSystemCommandName !== SYSTEM_COMMANDS.transmit;
+        const command = AIRCRAFT_COMMAND_MAP[callsignOrSystemCommandName];
```

Comment:
why not use `_has()` here?

Reply:
no longer needed, due to the rearrangement of the command map, and resulting adjustments to this method. should also read slightly clearer now as well, so that's also a plus!

done